### PR TITLE
Feature/kas 13 refactor and reorganzie leeze it

### PR DESCRIPTION
**Pull request by Harrisali93**
## Jira Story Link:
[KAS-13](https://leezeit.atlassian.net/browse/KAS-13?atlOrigin=eyJpIjoiNDUxOTM3M2VjMmI1NDViODgyZjk5ODgzZTVlMTM5MmIiLCJwIjoiaiJ9)

**Description**:
Refactor and Reorganize the files
Proper File names
Follow MVC Design Pattern
Enums for Storyboards

**How to test this **:
open up the project and check in Navigator Panel

### ScreenShots & Gifs:
Screen Shot 2022-09-18 at 01 26 42

### CODE REVIEW

File: LeezeIt/LeezeIt/Controller/LogInSignUpViewController.swift [hunk 1]
```diff
@@ -35,61 +34,55 @@ class ViewController: UIViewController {
...
 
     @objc func logIn(_ sender: UIButton) {
-        print("LogIn Button Tapped")
-        
+        let storyBoard = UIStoryboard(name:StoryBoardsID.LogIn.rawValue, bundle: nil)
+        let nextvc = storyBoard.instantiateViewController(withIdentifier: StoryBoardsID.LogIn.rawValue) as! LogInViewController
+       present(nextvc, animated: true)
     }
     
     @objc func signUP(_ sender: UIButton) {
```
Comment (arfhan786) [inline LeezeIt/LeezeIt/Controller/LogInSignUpViewController.swift]: p should be lower case

File: LeezeIt/LeezeIt/Controller/LogInViewController.swift [hunk 2]
```diff
@@ -7,7 +7,7 @@
 import Foundation
 import UIKit
 
-class PhoneNumberViewController:UIViewController{
+class LogInViewController:UIViewController{
```
Comment (arfhan786) [inline LeezeIt/LeezeIt/Controller/LogInViewController.swift]: revert the name change

File: LeezeIt/LeezeIt/Controller/LogInSignUpViewController.swift [hunk 3]
```diff
@@ -35,61 +34,55 @@ class ViewController: UIViewController {
...
     }
     
     func imageStackViewConstraint() {
-        leezeitImageStackView.translatesAutoresizingMaskIntoConstraints                                               = false
-        leezeitImageStackView.centerXAnchor.constraint(equalTo: parentStackView.centerXAnchor).isActive               = true
-        leezeitImageStackView.topAnchor.constraint(equalTo: parentStackView.bottomAnchor, constant: -10).isActive     = true
+        leezeitImageStackView.translatesAutoresizingMaskIntoConstraints = false
+        leezeitImageStackView.centerXAnchor.constraint(equalTo: parentStackView.centerXAnchor).isActive = true
+        leezeitImageStackView.topAnchor.constraint(equalTo: parentStackView.bottomAnchor, constant: -10).isActive = true
         leezeitImageStackView.addArrangedSubview(leezeitImage)
```
Comment (arfhan786) [inline LeezeIt/LeezeIt/Controller/LogInSignUpViewController.swift:75]: does it work if we dont put image in stack?

File: LeezeIt/LeezeIt/Controller/LogInViewController.swift [hunk 4]
```diff
@@ -7,7 +7,7 @@
 import Foundation
 import UIKit
```
Comment (arfhan786) [inline LeezeIt/LeezeIt/Controller/LogInViewController.swift]: we dont need import Foundation

File: LeezeIt/LeezeIt/Controller/LogInViewController.swift [hunk 5]
```diff
@@ -34,12 +34,14 @@ class PhoneNumberViewController:UIViewController{
...
         super.viewDidLoad()
-        
         fetchUserDefaultData()
  }
     
     @IBAction func nextButton(_ sender: Any) {
         UserDefaults.standard.set(phoneNumberText.text!, forKey: "userPhoneNumber")
+        let storyBoard = UIStoryboard(name:StoryBoardsID.Verification.rawValue, bundle: nil)
+        let nextvc = storyBoard.instantiateViewController(withIdentifier: StoryBoardsID.Verification.rawValue) as! VerificationViewController
+       present(nextvc, animated: true)
```
Comment (arfhan786) [inline LeezeIt/LeezeIt/Controller/LogInViewController.swift]: fix the indentation

File: LeezeIt/LeezeIt/Controller/SignUpViewController.swift [hunk 6]
```diff
@@ -7,7 +7,7 @@
 
 import UIKit
 
-class CreateAccountViewController: UIViewController, UITextFieldDelegate {
+class SignUpViewController: UIViewController, UITextFieldDelegate {
```
Comment (arfhan786) [inline LeezeIt/LeezeIt/Controller/SignUpViewController.swift]: move the UITextFieldDelegate in extension

File: LeezeIt/LeezeItTests/LogInViewControllerTests.swift [hunk 7]
```diff
@@ -8,9 +8,9 @@
 import XCTest
 @testable import LeezeIt
 
-class PhoneNumberViewControllerTests: XCTestCase {
+class LogInViewControllerTests: XCTestCase {
```
Comment (arfhan786) [inline LeezeIt/LeezeItTests/LogInViewControllerTests.swift]: revert this

File: LeezeIt/LeezeIt/Enums/Enums_StoryBoard.swift [hunk 8]
```diff
@@ -0,0 +1,15 @@
...
+//  Enums_StoryBoard.swift
+//  LeezeIt
+//
+//  Created by harris ali on 9/17/22.
+//
+
+import Foundation
+
+enum StoryBoardsID : String {
+    case LogInSignUp = "LogInSignUp"
```
Comment (arfhan786) [inline LeezeIt/LeezeIt/Enums/Enums_StoryBoard.swift]: maybe we should name all of these like LogInSignUpStoryBoard

File: LeezeIt/LeezeIt/Controller/SignUpViewController.swift (hunk 6 above)
Comment (Harrisali93) [reply LeezeIt/LeezeIt/Controller/SignUpViewController.swift]: Resolved

File: LeezeIt/LeezeItTests/LogInViewControllerTests.swift (hunk 7 above)
Comment (Harrisali93) [reply LeezeIt/LeezeItTests/LogInViewControllerTests.swift]: resolved

File: LeezeIt/LeezeIt/Enums/Enums_StoryBoard.swift (hunk 8 above)
Comment (Harrisali93) [reply LeezeIt/LeezeIt/Enums/Enums_StoryBoard.swift]: resolved

File: LeezeIt/LeezeIt/Controller/LogInViewController.swift (hunk 5 above)
Comment (Harrisali93) [reply LeezeIt/LeezeIt/Controller/LogInViewController.swift]: resolved

File: LeezeIt/LeezeIt/Controller/LogInViewController.swift (hunk 4 above)
Comment (Harrisali93) [reply LeezeIt/LeezeIt/Controller/LogInViewController.swift]: resolved

File: LeezeIt/LeezeIt/Controller/LogInViewController.swift (hunk 2 above)
Comment (Harrisali93) [reply LeezeIt/LeezeIt/Controller/LogInViewController.swift]: resolved

File: LeezeIt/LeezeIt/Controller/LogInSignUpViewController.swift (hunk 1 above)
Comment (Harrisali93) [reply LeezeIt/LeezeIt/Controller/LogInSignUpViewController.swift]: resolved

File: LeezeIt/LeezeIt/Controller/LogInSignUpViewController.swift (hunk 3 above)
Comment (Harrisali93) [reply LeezeIt/LeezeIt/Controller/LogInSignUpViewController.swift:75]: i would prefer stack for better constraints

**Review by arfhan786 — APPROVED** [review]

**Pull request merged**